addon/mocks/mutation.js

```diff
@@ -13,6 +14,10 @@ export const composeMockMutation = (getRecords, mapArgs) =>
   (db, options = {}, _, args, __, { fieldName, returnType }) => {
     returnType = unwrapNonNull(returnType);
 
+    if (returnType instanceof GraphQLList) {
```

Comment:
I'm not 💯 sure about using `GraphQLlist`. I'm not super familiar with graphql types but it seemed like the thing to use. I tried adding it in the `unwrapNonNull` util but that broke other tests, I'm guessing `.ofType` must be accessed in other parts of the app so adding it here keeps it scoped to mutations.

Reply:
This looks good to me.